packages/grid/x-data-grid/src/models/events/gridEventLookup.ts

```diff
@@ -168,7 +168,7 @@ export interface GridEventLookup
     params: GridCellParams<any, any, any, any>;
     event: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>;
   };
-  cellEditStop: { params: GridCellParams<any, any, any>; any; event: MuiBaseEvent };
+  cellEditStop: { params: GridCellParams<any, any, any>; any: any; event: MuiBaseEvent };
```

Comment:
```suggestion
  cellEditStop: { params: GridCellParams<any, any, any>; event: MuiBaseEvent };
```

I think this `any` is just a typo, the lookup should only have `params` and `event` properties